app/components/Showcases/Barter.jsx

```diff
@@ -78,13 +72,18 @@ export default class Barter extends Component {
                 to_barter: [{message: "", shown: false}],
                 escrow: [{message: "", shown: false}]
             },
-            proposal_fee: 0,
+            proposal_fee: {
```

Comment:
Why is this split in amount and real amount necessary?

Reply:
fixed